lang/src/gen/build.rs

```diff
@@ -57,6 +57,7 @@ fn codegen_for_contract_env(tokens: &mut TokenStream2, contract: &hir::Contract)
             use ink_core::env::{ContractEnv, EnvTypes};
 
             pub type AccountId = <ContractEnv<#env_types> as EnvTypes>::AccountId;
+            pub type AccountIndex = <ContractEnv<#env_types> as EnvTypes>::AccountIndex;
```

Comment:
What is the difference to AccountId?

Reply:
A lookup of `AccountIndex` -> `AccountId`. e.g. `node-runtime` has a lookup `u32` -> `Public` to resolve an address from an `Index`. See https://github.com/paritytech/substrate/blob/master/srml/balances/src/lib.rs#L417

Reply:
Ah got it, that's what an `Address` can also refer to instead of an `AccountId`.